goal_src/jak2/engine/game/task/task-arrow.gc

```diff
@@ -5,5 +5,337 @@
 ;; name in dgo: task-arrow
 ;; dgos: ENGINE, GAME
 
+;; +++task-arrow-flags
+(defenum task-arrow-flags
+  :type uint32
+  :bitfield #t
+  (task-arrow-flag-00 0)
+  (task-arrow-flag-01 1)
+  (task-arrow-flag-02 2)
+  (task-arrow-flag-03 3)
+  )
+;; ---task-arrow-flags
+
 ;; DECOMP BEGINS
 
+(let ((a0-0 (new 'static 'skeleton-group
+              :name "skel-task-arrow"
+              :extra #f
+              :info #f
+              :art-group-name "ctywide-arrow"
+              :bounds (new 'static 'vector :w 16384.0)
+              :texture-level #x6
+              :version #x7
+              )
+            )
+      )
+  (set! (-> a0-0 jgeo) 0)
+  (set! (-> a0-0 janim) 2)
+  (set! (-> a0-0 mgeo 0) 1)
+  (set! (-> a0-0 lod-dist 0) 4095996000.0)
+  (add-to-loading-level a0-0)
+  )
```

Comment:
this looks like it should have become a `defskelgroup`... any chance this is from an older version of the decompiler?